app/controllers/concerns/finder_top_result_ab_testable.rb

```diff
@@ -0,0 +1,28 @@
+module FinderTopResultAbTestable
+  CUSTOM_DIMENSION = 69
+
+  def self.included(base)
+    base.helper_method(
+      :finder_top_result_test_variant,
+      :show_top_result?
+    )
+    base.after_action :set_test_response_header
+  end
+
+  def finder_top_result_test
+    @finder_top_result_variant ||= GovukAbTesting::AbTest.new(
+      "FinderAnswerABTest",
+      dimension: CUSTOM_DIMENSION,
+      allowed_variants: %w(A B),
+      control_variant: "A"
+    )
+  end
+
+  def finder_top_result_test_variant
```

Comment:
The name is a bit too similar to `finder_top_result_test`, perhaps drop the `test` from the name?